village/src/App.js

```diff
@@ -31,8 +31,17 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <SmurfForm addSmurf={this.addSmurf} />
-        <Smurfs smurfs={this.state.smurfs} />
+        <nav className="nav">
+          <NavLink to="/">Home</NavLink>
```

Comment:
you can also utilize NavLink to show active styling to identfy which Nav you're on .